### PR TITLE
[Sage-944] Classnames Utility Proof of Concept

### PR DESCRIPTION
**Pull request by FuturaExtraBold**
## Description
Adds a classnames helper that converts an object to classes for styling.

## Related
#944

### CODE REVIEW

File: docs/app/views/examples/components/classnames_test/_preview.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<h3>Default</h3>
+<% cn = sage_classnames({
+  align: "center",
+  base_classes: "alpha bravo charlie",
```
Comment (philschanely) [inline docs/app/views/examples/components/classnames_test/_preview.html.erb:4]: Not opposed to this but technically one could put these in outside of our function. Might consider keeping ours focused just on what Sage offers?

**Comment by philschanely** [comment]
@FuturaExtraBold this is off to a great start. Is the new component just for testing now? I think this could be documented and demonstrated under the Helpers menu instead of Components.

File: docs/app/views/examples/components/classnames_test/_preview.html.erb (hunk 1 above)
Comment (FuturaExtraBold) [reply docs/app/views/examples/components/classnames_test/_preview.html.erb:4]: Yes. I will remove the `base_classes`.

**Comment by FuturaExtraBold** [comment]
> @FuturaExtraBold this is off to a great start. Is the new component just for testing now? I think this could be documented and demonstrated under the Helpers menu instead of Components.

@philschanely Yes. This is just for testing as I work. It should be one helper file when work is complete.

**Pull request closed without merging**